src/components/navbar/Navbar.js

```diff
@@ -4,13 +4,25 @@ import SearchIcon from '@mui/icons-material/Search';
 import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
 import Person2Icon from '@mui/icons-material/Person2';
 import { blue } from 'material-ui-colors';
+import { useNavigate } from 'react-router-dom';
+
 function Navbar() {
-    const pages = [{name: 'Soccer', path: 'soccer'}, {name: 'Basketball', path: '#news'}, , {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'},{name: 'Courses', path: '#news'}];
+
+    const pages = [{name: 'Soccer', path: 'soccer'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'}, {name: 'Basketball', path: '#news'},{name: 'Courses', path: '#news'}];
+    const navigate = useNavigate();
+
+    const handleCartClick = () => {
+        navigate('/cart'); 
+    };
+
+    const redirectToHome = () =>{
+        navigate('/')
+    }
 
     return (
         <div class="topnav">
             <div class="logo">
-                <a class="active" href="#home">Logo</a>
+             <img onClick={redirectToHome} style={{ width: '70px', height: '100px'}} src={require('../../assets/images/Group.png')}></img>
```

Comment:
I didn't find group.png and it did give us an error when we execute please add group.png

Reply:
fixed!